taggit/models.py

```diff
@@ -68,8 +68,11 @@ def save(self, *args, **kwargs):
             except IntegrityError:
                 pass
             # Now try to find existing slugs with similar names
-            slugs = set(Tag.objects.filter(slug__startswith=self.slug)
-                                   .values_list('slug', flat=True))
+            slugs = set(
+                self._default_manager
```

Comment:
Ough I missed @groovecoder's comment. I'm away from the keyboard right now so can't link to all the cases in django's source code easily, but the `model` he refers to is a model class, not an instance of a model class. I'm not sure what else to say other than that if you want to use a private API like this (as it makes sense) I'd strongly suggest to follow the precedence to reduce the chance for surprises.

> On 08 Aug 2015, at 16:32, Corey Farwell <notifications@github.com> wrote:
> 
> In taggit/models.py:
> 
> > @@ -68,8 +68,11 @@ def save(self, *args, **kwargs):
> >              except IntegrityError:
> >                  pass
> >              # Now try to find existing slugs with similar names
> > -            slugs = set(Tag.objects.filter(slug__startswith=self.slug)
> > -                                   .values_list('slug', flat=True))
> > +            slugs = set(
> > +                self._default_manager
> If a decision here ever gets established, feel free to open another PR with the fix
> 
> —
> Reply to this email directly or view it on GitHub.
>